site-manage/site-manage-impl/impl/src/java/org/sakaiproject/sitemanage/impl/PortalSubPageNavProviderImpl.java

```diff
@@ -0,0 +1,16 @@
+package org.sakaiproject.sitemanage.impl;
+
+import org.sakaiproject.portal.api.PortalSubPageNavProvider;
+
+public class PortalSubPageNavProviderImpl implements PortalSubPageNavProvider {
```

Comment:
this probably wasn't needed?

Reply:
Correct..... it doesn't have enough info to be able to implement this .... you need to know the parents and children of all sites the user has access to